Client/Extensions/Rewards/BraveRewardsExtensions.swift

```diff
@@ -40,9 +40,12 @@ extension BraveRewards {
             return
         }
         isCreatingWallet = true
-        ledger.createWalletAndFetchDetails { [weak self] success in
-            self?.isCreatingWallet = false
-            completion?()
+        startLedgerService {
+            guard let ledger = self.ledger else { return }
```

Comment:
Why do you unwrap it here not before the `startLedgerService` call?
Do we have to call this method no matter if ledger exists or not

Reply:
Yeah the start method will just complete immediately if it's already been started so it's better to just always call it to ensure it's always ready after